lang/cs/Org.Apache.REEF.Bridge.JAR/Org.Apache.REEF.Bridge.JAR.csproj

```diff
@@ -25,7 +25,18 @@ under the License.
     <AssemblyName>Org.Apache.REEF.Bridge.JAR</AssemblyName>
     <RestorePackages>true</RestorePackages>
     <BuildPackage>false</BuildPackage>
+    <TargetFrameworkVersion>v4.5.1</TargetFrameworkVersion>
+    <TargetFrameworkProfile />
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x64'">
```

Comment:
Again I see this chunk of xml:
<PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x64'">
    <Prefer32Bit>false</Prefer32Bit>
  </PropertyGroup>

So I think the change may be some spaces or something, not sure but the tag is still needed